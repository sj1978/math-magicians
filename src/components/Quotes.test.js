import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from './Quote';

describe('Quotes', () => {
  test('renders quote component', () => {
    const tree = renderer.create(<Quotes />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
