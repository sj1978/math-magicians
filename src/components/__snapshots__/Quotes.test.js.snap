// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes renders quote component 1`] = `
<section>
  <p
    className="quotes"
  >
    Pure mathematics is, in its way, the poetry of logical ideas. â€” Albert Einstein, German theoretical physicist
  </p>
</section>
`;
